.github/workflows/docker_build_tpls.yml

```diff
@@ -34,20 +34,20 @@ jobs:
       id: repo_name
```

Comment:
@chapman39 has the rest of the changes needed in this PR.

https://github.com/LLNL/serac/pull/879

Reply:
Thanks -- I got pulled in other directions. 
I'll check it out and update this one when I can.